client/blocks/simple-site-rename-form/index.jsx

```diff
@@ -122,6 +122,22 @@ export class SimpleSiteRenameForm extends Component {
 		const isDisabled =
 			! domainFieldValue || !! domainFieldError || domainFieldValue === currentDomainPrefix;
 
+		if ( ! currentDomain.currentUserCanManage ) {
+			return (
+				<div className="simple-site-rename-form simple-site-rename-form__only-owner-info">
+					<Gridicon icon="info" />
+					{ translate( 'Only the site owner%(ownerInfo)s can edit this domain name.', {
```

Comment:
This can be a bit hairy to translate. Maybe a better variable text can help, for example `Only the site owner%(optionalOwnerName)s can edit this domain name.`?

Reply:
I think we could actually keep the space: `Only the site owner %(optionalOwnerName)s can edit this domain name.` It will make it less confusing to translators and a double-space in HTML is irrelevant anyway.

Reply:
On a second thought I think we should really have two translatable texts: one without and one with variable. It works in English to potentially omit a name but this might not work in other languages.

Reply:
Thanks for the suggestion, @akirk. I went through a similar thought process & almost had two strings.  Will change.